requirements-optional-26.txt

```diff
@@ -0,0 +1,10 @@
+# https://bitbucket.org/logilab/astroid/issue/76/python26-compatibility
```

Comment:
Wow, these many requirements for python 2.6! Guess we need to get Travis aware of them when running tests?